src/current/_includes/releases/v23.1/v23.1.25.md

```diff
@@ -0,0 +1,80 @@
+    - [`ALTER BACKUP SCHEDULE`]({% link v23.1/alter-backup-schedule.md %})
+    - [`ALTER BACKUP`]({% link v23.1/alter-backup.md %})
+    - [`SHOW BACKUP`]({% link v23.1/show-backup.md %})
+    - [`CREATE CHANGEFEED`]({% link v23.1/create-changefeed.md %})
+    - [`CREATE SCHEDULE FOR CHANGEFEED`]({% link v23.1/create-schedule-for-changefeed.md %})
+    - [`ALTER CHANGEFEED SET {sink}`]({% link v23.1/alter-changefeed.md %})
+    - [`IMPORT`]({% link v23.1/import.md %})
+    - [`EXPORT`]({% link v23.1/export.md %})
+    - [`CREATE EXTERNAL CONNECTION`]({% link v23.1/create-external-connection.md %})
+    - [`COPY`]({% link v23.1/copy-from.md %})
+
+    [#127537][#127537]
+- The cluster setting [`server.jwt_authentication.issuers`]({% link v23.1/sso-sql.md %}) can now take multiple values to support various kinds of providers and their mapped JWKS URI. This can be set to one of the following values:
+    - Simple string that Go can parse as a valid issuer URL: `'https://accounts.google.com'`.
+    - String that can be parsed as a valid JSON array of issuer URLs list: `['example.com/adfs','https://accounts.google.com']`.
+    - String that can be parsed as a valid JSON and deserialized into a map of issuer URLs to corresponding JWKS URIs. In the third case, CockroachDB will override the JWKS URI present in the issuer's well-known endpoint: `'{ "issuer_jwks_map": { "https://accounts.google.com": "https://www.googleapis.com/oauth2/v3/certs", "example.com/adfs": "https://example.com/adfs/discovery/keys" } }'`.
+
+    When `issuer_jwks_map` is set, CockroachDB directly uses the JWKS URI to get the key set. In all other cases where [`server.jwt_authentication.jwks_auto_fetch.enabled`]({% link v23.1/sso-sql.md %}#cluster-settings) is set CockroachDB obtains the JWKS URI first from issuer's well-known endpoint and then uses this endpoint. [#128751][#128751]
+
+<h3 id="v23-1-25-sql-language-changes">SQL language changes</h3>
+
+- Added the `sql.auth.grant_option_inheritance.enabled` [cluster setting]({% link v23.1/cluster-settings.md %}). The default value is `true`, which results in behavior that matches the existing behavior of CockroachDB. When set to `false`, the [`GRANT OPTION`]({% link v23.1/grant.md %}) is not inherited via role membership, which will prevent the descendant roles from being able to grant the privilege to others. The privilege itself is still inherited via role membership. [#126862][#126862]
+- Added the `sql.auth.grant_option_for_owner.enabled` [cluster setting]({% link v23.1/cluster-settings.md %}). The default value is `true`, which results in behavior that matches the existing behavior of CockroachDB. When set to `false`, then the [`GRANT OPTION`]({% link v23.1/grant.md %}) is not implcitly given to the owner of an object. The object owner still implicitly has all privileges on the object, just not the ability to grant them to other users. [#127005][#127005]
```

Comment:
maybe add the direct setting anchor? `#setting-sql-auth-grant-option-for-owner-enabled` (checked and these are both there)